### PR TITLE
Lambda ui for var def and iterative functions.

### PR DESCRIPTION
**Pull request by manuranga**
works for basic iterative functions. eg:-
```
function main (string[] args) {
    x = a.apply(function(){
        a = b;
        int s = b;
   });
}

```

### CODE REVIEW

**Comment by manuranga** [comment]
@maheeka @keizer619 
You should render a normal assignment jsx, and it should render the lambda box with it.

File: modules/web/js/ballerina/visitors/dimension-calculator/assignment-statement-dimension-calculator-visitor.js [hunk 1]
```diff
@@ -62,10 +71,20 @@ class AssignmentStatementDimensionCalculatorVisitor {
      * @memberOf AssignmentStatementDimensionCalculatorVisitor
      * */
     endVisit(node) {
-        util.populateSimpleStatementBBox(node.getStatementString(), node.getViewState());
-        if (node.getFactory().isConnectorInitExpression(node.getChildren()[1])) {
+        const viewState = node.getViewState();
+        util.populateSimpleStatementBBox(node.getStatementString(), viewState);
+        const child = node.getChildren()[1];
```
Comment (maheeka) [inline modules/web/js/ballerina/visitors/dimension-calculator/assignment-statement-dimension-calculator-visitor.js]: can we use getRightExpression here

File: modules/web/js/ballerina/visitors/dimension-calculator/assignment-statement-dimension-calculator-visitor.js [hunk 2]
```diff
@@ -51,7 +53,14 @@ class AssignmentStatementDimensionCalculatorVisitor {
      *
      * @memberOf AssignmentStatementDimensionCalculatorVisitor
      * */
-    visit() {
+    visit(node) {
+        const child = node.getChildren()[1];
```
Comment (maheeka) [inline modules/web/js/ballerina/visitors/dimension-calculator/assignment-statement-dimension-calculator-visitor.js]: getRightExpression can be used here

File: modules/web/js/ballerina/visitors/position-calc/assignment-statement-position-calc.js [hunk 3]
```diff
@@ -61,8 +62,15 @@ class AssignmentStatementPositionCalcVisitor {
      *
      * @memberOf AssignmentStatementPositionCalcVisitor
      * */
-    visit() {
+    visit(node) {
         log.debug('visit AssignmentStatementPositionCalc');
+        const child = node.children[1];
```
Comment (maheeka) [inline modules/web/js/ballerina/visitors/position-calc/assignment-statement-position-calc.js]: getRightExpression here

File: modules/web/js/ballerina/visitors/dimension-calculator/assignment-statement-dimension-calculator-visitor.js (hunk 2 above)
Comment (maheeka) [inline modules/web/js/ballerina/visitors/dimension-calculator/assignment-statement-dimension-calculator-visitor.js]: Can we move this logic from visit()?

**Comment by manuranga** [comment]
removed `getChildren()[1]` and `getChildren()[0]` removing visit logic needs some refactoring, I'll send that with next PR.

**Pull request merged**